src/main/java/com/epam/java/pdp/figures/Shape.java

```diff
@@ -0,0 +1,65 @@
+package com.epam.java.pdp.figures;
+import com.epam.java.pdp.figures.Point;
+
+public class Shape {
+    static int x1 = 1;
+    static int x2 = 2;
+    static int x3 = 3;
+    static int y1 = 2;
+    static int y2 = 1;
+    static int y3 = 3;
+    static Point p1 = new Point(x1, y1);
```

Comment:
Please remove all static specifiers except the declaration of main method (as we agreed before)

Reply:
done